octopoes/tests/integration/test_api_connector.py

```diff
@@ -86,6 +89,56 @@ def test_bulk_operations(octopoes_api_connector: OctopoesAPIConnector, valid_tim
         assert bulk_hostnames[hostname.reference].scan_profile is not None
 
 
+def test_bulk_reports(app_settings: Settings, octopoes_api_connector: OctopoesAPIConnector, valid_time: datetime):
+    filters = []
+    reports = []
+
+    for client in ["test1", "test2", "test3"]:
+        xtdb_client = get_xtdb_client(str(app_settings.xtdb_uri), client)
+        xtdb_client.create_node()
+
+        octopoes_api_connector.client = client
+        recipe = ReportRecipe(
+            report_type="concatenated-report",
+            recipe_id=uuid.uuid4(),
+            report_name_format="test",
+            cron_expression="* * * *",
+            input_recipe={},
+            asset_report_types=[],
+        )
+        report = Report(
+            name=f"report-{client}",
+            date_generated=valid_time,
+            organization_code="code",
+            organization_name="name",
+            organization_tags=["tag1", "tag2"],
+            data_raw_id="raw",
+            observed_at=valid_time,
+            reference_date=valid_time,
+            report_recipe=recipe.reference,
+            input_oois=[],
+            report_type="concatenated-report",
+        )
+        octopoes_api_connector.save_declaration(Declaration(ooi=recipe, valid_time=valid_time))
+        octopoes_api_connector.save_declaration(Declaration(ooi=report, valid_time=valid_time))
```

Comment:
Duplicate statement?

Reply:
One for the report, one for the recipe :)